src/ecs-additions/ecs-logs-collector.sh

```diff
@@ -0,0 +1,723 @@
+#!/usr/bin/env bash
+#
+# Copyright 2016-2018 Amazon.com, Inc. or its affiliates.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#    http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+#
+# - Collects Docker daemon and Amazon ECS Container Agent logs on Amazon Linux,
+#   Redhat 7, Debian 8.
+# - Collects general operating system logs.
+# - Optional ability to enable debug mode for the Docker daemon and Amazon ECS
+#   Container Agent on Amazon Linux variants, such as the Amazon ECS-optimized
+#   AMI. For usage information, see --help.
+
+export LANG="C"
+export LC_ALL="C"
+
+# Collection configuration
+
+# curdir is the working root of collection.
+curdir="$(dirname "$0")"
+# collectdir is where all collected informaton is placed under. This
+# services as the top level for this script's operation.
+readonly collectdir="${curdir}/collect"
+# pack_name is the name of the resulting tarball. This will generally
+# be collect-i-ffffffffffffffffff, where i-ffffffffffffffffff is the
+# instance id.
+pack_name="collect"
```

Comment:
does the pack_name need to include the instance id to prevent namespace collisions?